### PR TITLE
Update homepage text aug 2023

### PR DESCRIPTION
**Pull request by rasummer**
@ennru this updates the Akka.io homepage with BSL language in the blue hero of the homepage. The content that originally featured in the blue area has been moved down into the section below. Request for these changes came from James Townley. 

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,7 +9,36 @@
 {% include lightbendBanner.html %}
 {% include topbar.html %}
 
-<section class="wrapper hero">
+<section class="wrapper hero-bsl">
+    <div class="row">
+        <div id="hero-bsl-left-col" class="sixcol">
+            <h2 class="bright">Do You Have Your Akka BSL License?</h2>
+            <p>A license ensures the version of Akka (beginning with v2.7 released Sept 2022) you
```
Comment (ennru) [inline index.html:16]:
```suggestion
            <p>A license ensures the version of Akka (beginning with version 2.7, released in September 2022) you
```
The sentence seems a little unnecessarily complicated.

File: index.html [hunk 2]
```diff
@@ -9,7 +9,36 @@
...
+                are running in production has all of the latest known bug fixes, vulnerabilities patched, and maintains
+                compliance with SOC 2 and NIST 800-53. </p>
+            <p>Licensing allows for production access to all Akka’s new features like <strong>Brokerless pub/sub</strong> and the upcoming release of
+                <strong>Akka Edge</strong> in late 2023.</p>
```
Comment (ennru) [inline index.html:20]:
```suggestion
            <p>Licensing allows for production access to all Akka’s new features like <strong>Brokerless pub/sub</strong> and the upcoming release of
                <strong>Akka Edge</strong> in late 2023.</p>
```
I might be better not to date this text, e.g. by listing existing features instead.

File: index.html [hunk 3]
```diff
@@ -9,7 +9,36 @@
...
+            <p>More information can be found in the <a href="https://www.lightbend.com/akka/license-faq" target="_blank">BSL License FAQ</a> or <a href="mailto:akka-license@lightbend.com?subject=Akka License">contact Lightbend</a> for a License or questions. </p>
```
Comment (ennru) [inline index.html:21]:
```suggestion
            <p>More information can be found in the <a href="https://www.lightbend.com/akka/license-faq" target="_blank">BSL License FAQ</a> or <a href="https://www.lightbend.com/contact" target="_blank">contact Lightbend</a> for a License or questions. </p>
```
Trying to reduce spam.

File: index.html [hunk 4]
```diff
@@ -9,7 +9,36 @@
...
+
+        </div>
+        <div class="onecol">&nbsp;</div>
+        <div id="hero-bsl-right-col" class="fivecol">
+            <p><strong>LICENSING GUIDELINES ARE AS FOLLOWS:</strong></p>
+            <ul>
+                <li><strong>Development and Testing</strong> - No charge, non-production usage does not require a license</li>
+                <li><strong>Revenue under $25M</strong> - FREE Startup License for production use of Akka </li>
+                <li><strong>Revenue more than $25M</strong> - choice of Production licenses</li>
```
Comment (ennru) [inline index.html:30]:
```suggestion
                <li><strong>Revenue under 25 mUSD</strong> - FREE Startup License for production use of Akka </li>
                <li><strong>Revenue more than 25 mUSD</strong> - choice of Production licenses</li>
```

**Review by ennru — APPROVED** [review]
A few suggestions but fine if you just want to get it done.

File: index.html (hunk 4 above)
Comment (patriknw) [reply index.html:30]: Is "mUSD" how it's normally written? Looks funny to me, google mostly finds "MUSD".

File: index.html (hunk 2 above)
Comment (rasummer) [reply index.html:20]: How about we just pull 'in late 2023'  ?

**Comment by rasummer** [comment]
Changes made:

image

File: index.html (hunk 1 above)
Comment (rasummer) [reply index.html]: Change commited

**Review by ennru — APPROVED** [review]
LGTM.

**Comment by rasummer** [comment]
@ennru is it good to merge despite the failed link validator?

**Comment by ennru** [comment]
> @ennru is it good to merge despite the failed link validator?

Yes, Ok to merge.

It complains with

## HTTP failure response
https://gist.github.com/johanandren/a5b9e4f88fc9fc6ce56dac10bb81a50e status -1 javax.net.ssl.SSLHandshakeException: No subject alternative DNS name matching gist.github.com found.
 - blog/article/2016/08/25/simple-sink-source-with-graphstage.html

https://doc.akka.io/docs/akka-persistence-spanner/current/ status 404 Not Found
 - blog/news/2021/08/19/akka-2.6.16-released.html

https://repo.akka.io/releases status 404 Not Found
 - blog/news/2012/03/06/akka-20-released.html
 - blog/news/2012/04/13/akka-2.0.1-released.html
 - ... 7 more


## Non-https pages
http://developer.lightbend.com/guides/akka-quickstart-java
 - try-akka/index.html

http://developer.lightbend.com/guides/akka-quickstart-scala
 - try-akka/index.html

**Pull request merged**